SingularityUI/app/templates/tasksTable/tasksBase.hbs

```diff
@@ -2,19 +2,19 @@
     <div class="col-md-12">
         <ul class="nav nav-pills">
             <li {{#ifEqual tasksFilter "active"}}class="active"{{/ifEqual}}>
-                <a href="{{appRoot}}/tasks">Active</a>
+                <a href="{{appRoot}}/tasks" class='has-tooltip' title="Currently running tasks" data-toggle='tooltip'>Active</a>
             </li>
             <li {{#ifEqual tasksFilter "scheduled"}}class="active"{{/ifEqual}}>
-                <a href="{{appRoot}}/tasks/scheduled">Scheduled</a>
+                <a href="{{appRoot}}/tasks/scheduled" class='has-tooltip' title="Tasks that will run when the time comes" data-toggle='tooltip'>Scheduled</a>
             </li>
             <li {{#ifEqual tasksFilter "cleaning"}}class="active"{{/ifEqual}}>
-                <a href="{{appRoot}}/tasks/cleaning">Cleaning</a>
+                <a href="{{appRoot}}/tasks/cleaning" class='has-tooltip' title="Tasks that are cleaning" data-toggle='tooltip'>Cleaning</a>
```

Comment:
Technically correct, but maybe for easier to read language something like, 'Tasks that have been asked to shut down'